src/components/ReduxAppWrapper/store/reducers/globals.js

```diff
@@ -11,6 +11,13 @@ const initialState = {
   isNavigationDisabled: false,
   isFullScreen: false,
   deviceHasCameraSupport: false,
+  urls: {
```

Comment:
[Minor] Not sure if having mapping with not consistent naming is good - in the future you might get lost in which environment variable is what on the app.

Reply:
Good point! Will discuss with team to agree on a consistent naming.